### PR TITLE
remove all require's when possible

### PR DESCRIPTION
**Pull request by erwinmombay**
in preparation for no-undef

### CODE REVIEW

**Comment by erwinmombay** [comment]
@mkhatib please review

File: extensions/amp-analytics/0.1/test/test-amp-analytics.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ import {
 } from '../../../../src/service/url-replacements-impl';
 import * as sinon from 'sinon';
 
+/* global require: false */
```
Comment (mkhatib) [inline extensions/amp-analytics/0.1/test/test-amp-analytics.js]: Is this for linting? Would this disable this for the entire file or just the following line?

**Comment by mkhatib** [comment]
LGTM w one question

File: extensions/amp-analytics/0.1/test/test-amp-analytics.js (hunk 1 above)
Comment (erwinmombay) [inline extensions/amp-analytics/0.1/test/test-amp-analytics.js]: it would be starting from here until the end of the file

Comment (mkhatib) [inline extensions/amp-analytics/0.1/test/test-amp-analytics.js]: Can you disable it after this line? Doesn't seem we require anything else.

Comment (erwinmombay) [inline extensions/amp-analytics/0.1/test/test-amp-analytics.js]: yeah i can do that, but will probably do it after everything else is cleaned up. hard to verify with all the lint noise

Comment (mkhatib) [inline extensions/amp-analytics/0.1/test/test-amp-analytics.js]: sgtm

**Pull request merged**